pkg/sfu/audiolevel.go

```diff
@@ -0,0 +1,91 @@
+package sfu
+
+import (
+	"sort"
+	"sync"
+)
+
+type audioStream struct {
+	id    string
+	sum   int
+	total int
+}
+
+type audioLevel struct {
+	sync.RWMutex
+	threshold uint8
+	streams   []*audioStream
```

Comment:
Should streams be a map? It would make `observe` and `removeStream` simpler (and faster). It might make `calc` a little more complex though.